### ISSUE TITLE
ProtonMail offline, Bridge alternative and more

### ISSUE THREAD

**Issue by vinyll** [scastiel/protonmail-export#17]
ProtonMail has released [ProtonMail Bridge](https://protonmail.com/bridge/) a few month ago to the public.
That product brings great benefits:
- offline contents
- local backup
- searchable emails
- easy to transfer through different email address
- …

This gives Protonmail a technical beter position than Tutanota and permits to compete with Gmail.

However the bridge is only reserved to paid users. I'm OK to pay for that service, but most people I would advocate ProtonMail to would refrain from that.

Protonmail-export solves a  couple of these points, like local backup and makes the rest feasible.
It is still technical to run and don't permit browse emails as easily as a mailbox.

My point here is to offer a solution that will permit a non power user to use ProtonMail offline; ideally as the Bridge.
I'm currently using protonmail-export and [pmpkpe](https://github.com/kantium/pmpkpe) and it feel like we could do something to make the whole process easy.

Working together with some motivated people, I believe we could enhance what ProtonMail is offering.
The dream option would be to offer an opensource Bridge, but is it feasible?
Alternative options would be to have a folder that would sync locally and that would be read just like an inbox in a email client.
Another approach would be to join [the unofficial ProtonMail Desktop](https://github.com/protonmail-desktop/application) and build in a local copy with export and offline mode.

I'm writing here, in protonmail-export GitHub, because I see this brick as the core solution and therefore the most able people to collaborate on such a project.

Looking forward to discuss and share ideas around this.

**Comment by vinyll** [comment]
Seems like there's an audiance around this: https://protonmail.uservoice.com/forums/284483-feedback/suggestions/32549443-desktop-bridge-for-free-users

**Comment by vladimiry** [comment]
Offline content cache and mails export is supported by [this](https://github.com/vladimiry/email-securely-app) desktop app, see beta releases [here](https://github.com/vladimiry/email-securely-app/releases).

**Comment by vinyll** [comment]
This EmailSecurely App looks very promising.
I've manage to get my Protonmail account working easily, it's quite intuitive.
I've switched off my connection and offline version was working as hoped (even though it started with a huge connection error popup).
Search is not implemented yet, but the offline feature is a huge thing!
Thanks for sharing that (art-)work @vladimiry!

**Comment by vladimiry** [comment]
>  Search is not implemented yet, but the offline feature is a huge thing!

Yes, search is not implemented yet. For now, you can export messages to EML files (already implemented since v2.0.0-beta.4), then import to any mail client and search there.

**Comment by vladimiry** [comment]
> even though it started with a huge connection error popup

@vinyll I believe this issue got resolved with this release https://github.com/vladimiry/email-securely-app/releases/tag/v2.0.0-beta.10
